src/app/core/view/dynamic-components-map.ts

```diff
@@ -38,10 +38,10 @@ export const DYNAMIC_COMPONENTS_MAP = new Map<string, any>([
   ["Form", FormComponent],
   ["DisplayDate", DisplayDateComponent],
   ["DisplayText", DisplayTextComponent],
-  ["DisplayTick", DisplayTickComponent],
+  ["DisplayCheckmark", DisplayCheckmarkComponent],
   ["ChildBlock", ChildBlockComponent],
   ["ChildBlockList", ChildBlockListComponent],
-  ["ListAttendance", ListAttendanceComponent],
+  ["ListAttendance", RecentAttendanceBlocksComponent],
```

Comment:
Can we rename it here as well? 
```suggestion
  [" RecentAttendanceBlocks", RecentAttendanceBlocksComponent],
```